toolkits/global/packages/global-pagination/README.md

```diff
@@ -0,0 +1,52 @@
+# Global Pagination
+
+A component for the controls of pagination.
+
+## Branding
+
+To include `global-pagination` component currently uses the `DEFAULT` brand only.
+
+## Example
+
+#### HTML
+```html
+<nav>
+    <ul class="c-pagination">
+        <li class="c-pagination__item">
+            <span class="c-pagination__link c-pagination__link--disabled">
+                <svg class="c-icon u-margin-right-xs c-pagination__icon" aria-hidden="true">
```

Comment:
remove `u-margin-right-xs`. This may not be available and is a product-specific decision

Reply:
Done!